network-api/networkapi/templates/pages/buyersguide/base.html

```diff
@@ -84,15 +84,25 @@
     {% endblock %}
   </div>
 
-  {% block hero %}{% endblock %}
+  <div class="">
+    <div class="tw-relative">
+      <div class="tw-relative tw-z-10">
```

Comment:
Do we need the wrapper without a class and the two nested `div`s with `tw-relative`?

Reply:
Yes, we need this 100%, else the shapes will overlap will existing content that has CSS transformations applied since that effects the  stacking order on the homepage. Since we are using gsap for a few things, I would recommend we keep this part.

Reply:
@fessehaye What do you think about adding some comment to explain this? It not immediately obvious why it is needed, at least to me. 

I also tired without `<div class="">` locally and it seemed to work, but maybe I am missing something.

Reply:
Oh I misread this, we need the two relative classes but we don't need the empty class div afaik. That was just a artifact of removing `underneath-screen-overlay`. I can add a comment about the two relative classes though